Redb.OBAC.EF/BL/ObjectStorage.cs

```diff
@@ -729,8 +729,8 @@ public async Task<IEnumerable<TreeNodeInfo>> GetTreeSubnodesDeep(Guid treeId, in
             if (startingNodeId.HasValue)
             {
                 // start from root
-                var qry = @"with recursive nodes(id, parent_id, external_id_int, external_id_str, inherit_parent_perms, owner_user_id) as (
-            select id, parent_id, external_id_int, external_id_str, inherit_parent_perms, owner_user_id
+                var qry = $"with {(ctx.DbType!=DbType.MsSql? "recursive":"" )} nodes(id, parent_id, external_id_int, external_id_str, inherit_parent_perms, owner_user_id) as ("+
```

Comment:
Would be better not to use dbtype property checking across the code, but rather implement a polymorpic method (e.g. GetTreeSubnodesDeepQuery within int *DbContext hive)